doc/ko/README.md

```diff
@@ -269,10 +354,13 @@ flame-graph 형식의 결과 또한 지원한다. 해당 데이터는 `uftrace d
 
 제약사항
 ===========
-- 리눅스에서 실행되는 네이티브 C/C++ 사용자 프로그램에 대해서만 사용 가능하다.
-- 이미 실행 중인 프로세스의 추적은 *불가능*하다.
+- 리눅스에서 실행되는 네이티브 C/C++/Rust/Python 사용자 프로그램에
```

Comment:
In the ReadMe.md "It can trace a C/C++/Rust/Python application on Linux and Android." is written.
How about "리눅스와 안드로이드에서 실행되는 네이티브 C/C++/Rust/Python 사용자 프로그램에"?

Reply:
"네이티브" 라는 단어도 빼는게 나을것 같습니다.